packages/ds-component/src/types.ts

```diff
@@ -683,10 +691,22 @@ export interface ObjectTables {
   [TableType.SUMMARY]?: ObjectRow[];
 }
 
+export interface DateRange {
```

Comment:
Nit instead of calling this `DateRange` I think we should call it `DateRanges` since it's a single object that has all DateRanges